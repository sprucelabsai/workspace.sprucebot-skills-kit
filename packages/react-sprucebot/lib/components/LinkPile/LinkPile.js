'use strict';

Object.defineProperty(exports, "__esModule", {
	value: true
});

var _styledComponents = require('styled-components');

var _styledComponents2 = _interopRequireDefault(_styledComponents);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var LinkPile = (0, _styledComponents2.default)('div').attrs({
	className: 'link__pile'
}).withConfig({
	displayName: 'LinkPile',
	componentId: 'sc-173mxr2-0'
})(['']);

exports.default = LinkPile;