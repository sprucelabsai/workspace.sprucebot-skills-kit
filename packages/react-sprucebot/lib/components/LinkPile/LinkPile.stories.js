'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _react3 = require('@storybook/react');

var _addonInfo = require('@storybook/addon-info');

var _react4 = require('@storybook/addon-knobs/react');

var _storybookReadme = require('storybook-readme');

var _LinkPile = require('./LinkPile');

var _LinkPile2 = _interopRequireDefault(_LinkPile);

var _LinkPile3 = require('./LinkPile.md');

var _LinkPile4 = _interopRequireDefault(_LinkPile3);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var stories = (0, _react3.storiesOf)('LinkPile', module);
stories.addDecorator(_react4.withKnobs);

stories.add('Interactive', (0, _storybookReadme.withReadme)(_LinkPile4.default, function () {
  return _react2.default.createElement(_LinkPile2.default, null);
}));