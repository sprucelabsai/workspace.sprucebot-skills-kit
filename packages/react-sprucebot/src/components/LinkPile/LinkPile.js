import styled from 'styled-components'

const LinkPile = styled.div.attrs({
	className: 'link__pile'
})``

export default LinkPile
