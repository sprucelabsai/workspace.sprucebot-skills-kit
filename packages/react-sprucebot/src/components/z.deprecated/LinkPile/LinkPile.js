import React from 'react'

const LinkPile = ({ className, ...props }) => {
	return <div {...props} className={`link__pile ${className || ''}`} />
}

export default LinkPile
