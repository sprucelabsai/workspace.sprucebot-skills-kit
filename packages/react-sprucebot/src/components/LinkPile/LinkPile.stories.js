import React from 'react'
import { storiesOf } from '@storybook/react'
import { withInfo } from '@storybook/addon-info'
import { withKnobs } from '@storybook/addon-knobs/react'
import { withReadme } from 'storybook-readme'
import Button from '../Button/Button'
import LinkPile from './LinkPile'
import readme from './LinkPile.md'

const stories = storiesOf('LinkPile', module)
stories.addDecorator(withKnobs)

stories.add(
	'Interactive',
	withReadme(readme, () => (
		<LinkPile>
			<Button>Button 1</Button>
			<Button>Button 2</Button>
			<Button>Button 3</Button>
		</LinkPile>
	))
)
