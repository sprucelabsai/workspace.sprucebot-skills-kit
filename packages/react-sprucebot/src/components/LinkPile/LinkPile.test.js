import React from 'react'
import LinkPile from './LinkPile'
import { shallow } from 'enzyme'

describe('LinkPile tests', () => {
	it('Should match the snapshot', () => {
		const renderedComponent = shallow(<LinkPile />)

		expect(renderedComponent).toMatchSnapshot()
	})
})
