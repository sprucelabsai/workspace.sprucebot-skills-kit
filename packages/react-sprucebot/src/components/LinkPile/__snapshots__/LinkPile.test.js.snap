// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="LinkPile-abktjg-0 link__pile fwfXpR"
/>
`;
